packages/components/src/Dropzone/sub-components/DropzoneIcon/DropzoneIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react"
+import classNames from "classnames"
+import { Icon } from "@kaizen/component-library"
+import uploadFileSvgIcon from "@kaizen/component-library/icons/import.icon.svg"
+import { AcceptedFileIcon } from "../SelectedFileMessage/SelectedFileMessage"
+
+const UploadFileIcon = ({ isHover = false }: { isHover?: boolean }): JSX.Element => (
+  <span className={classNames({
+    "inline-block text-purple-800/70 mb-8 transition-transform -translate-y-0": true,
+    "!text-blue-500 !-translate-y-6": isHover
+  })}>
```

Comment:
You can pass `classNames()` into `Icon` via the `classNameOverride`, so you don't need this span